src/Components/Views/Searchs/Searchs.js

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState } from "react";
+
+//importacion de librerias externas
+import { Container, Row, Col } from "react-bootstrap";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import InfiniteScroll from "react-infinite-scroll-component";
+import { useParams } from "react-router-dom";
+
+//importacion de actions de redux
+
+//importacion de state de redux
+
+//importacion de reducer de redux
+import {
+	getSeriesError,
+	getSeriesPending,
+	getSeriesSearch,
+} from "../../../Redux/Reducer/index";
+
+//importacion de componentes
+import CardSerie from "../../Components/CardSerie/CardSerie";
+import { fetchSearchSeries } from "../../../Requests/Requests";
+
+/**
+ * Vista de una busqueda de serie en la aplicacion
+ */
+const Searchs = ({ fetchSeries, series }) => {
```

Comment:
I recommend you to use the next style guide: function Searchs(props). Don't forget the prop-types.